GraceChurch/src/theme/theme.js

```diff
@@ -113,9 +118,34 @@ const overlays = {
 //   ...propOverrides,
 // };
 const overrides = {
+  H1: {
+    fontFamily: typography.sans.light.default,
+  },
+  H2: {
+    fontFamily: typography.sans.light.default,
+    fontSize: typography.baseFontSize * 1.5,
```

Comment:
Not sure if this is exactly right but you would probably want to use `rem` here
```suggestion
    fontSize: helpers.rem(1.5),
```

Reply:
👍 fixed